css/style.css

```diff
@@ -294,13 +293,35 @@ figure{
   padding: 30px;
 }
 
+footer{
+  grid-column: 1 ;
```

Comment:
Pay attention to whitespaces in your document. See that you don't have redundant spaces or missing spaces.